microsoft-edge/devtools/whats-new/2025/05/devtools-136.md

```diff
@@ -27,10 +27,13 @@ Each DOM node that has a semantic issue displays a red wavy line, and has a tool
 
 ![DOM adorner in Elements](./devtools-136-images/dom-adorner.png)
 
+<!-- todo: checkbox gone from Stable 142/ Canary 144, delete?
```

Comment:
ok to delete old info about turning on Exper. Feat. checkbox?  
is the remaining section valid, then?  the feature is now present w/o setting the checkbox?

@captainbrosset @leahmsft 

Rendered article section:
* **What's New in DevTools (Microsoft Edge 136)** > **DOM tree in Elements tool highlights semantic errors**
   * https://review.learn.microsoft.com/microsoft-edge/devtools/whats-new/2025/05/devtools-136?branch=pr-en-us-3636#dom-tree-in-elements-tool-highlights-semantic-errors